ArmTemplates/azure-maps.md

```diff
@@ -0,0 +1,21 @@
+# Azure Maps
+
+Creates an Azure Maps Account
+
+## Paramaters
+
+azureMapName: (required) string
```

Comment:
Update to match change to template.  The service is called Azure Maps rather than Map so all references in the docs should refer to that